editoast/editoast_schemas/src/infra/railjson.rs

```diff
@@ -17,6 +17,10 @@ use super::TrackSection;
 
 pub const RAILJSON_VERSION: &str = "3.4.13";
 
+pub fn min_version_prefix() -> String {
+    RAILJSON_VERSION.split('.').next().unwrap().to_owned() + "."
+}
```

Comment:
I'm not sure that we need to reconstruct a `String` with a dot `.`. How about this function is used like this instead

```rust
fn major_version(version: &str) -> &str {
  version.split_once('.').unwrap()
}

// And then use it like
if major_version(version) != major_version(RAILJSON_VERSION)
```